### DISCUSSION TITLE
Disable local login after enabling SSO for SFTPGO

### DISCUSSION THREAD

**Discussion by fdb3** [drakkan/sftpgo#1522] [Q&A]
Hi,

I have an SFTPGO server and managed to get the SSO working. However, in order for this to work I've had to make an admin account in sftpgo manually as well. THis required a password, but I don't want admins to be able to log in with this local password. Is it possible to disable local login? Can this be done for specific users? Or only generically? Do you have an example of how I can do this?

Kind regards,

Fiona

**Comment by useronkel** [comment]
Hi.
If you are using OIDC, perhaps you might have a look at
```
enabled_login_methods, integer.
Defines the login methods available for the WebAdmin and WebClient UIs.
0 means any configured method: username/password login form and OIDC, if enabled.
1 means OIDC for the WebAdmin UI.
2 means OIDC for the WebClient UI.
4 means login form for the WebAdmin UI.
8 means login form for the WebClient UI. You can combine the values. 
For example 3 means that you can only login using OIDC on both WebClient and WebAdmin UI.
Default: 0.
```
You can find it at the json or the .env file
Settings in sftpgo.env: SFTPGO_HTTPD__BINDINGS__0__ENABLED_LOGIN_METHODS
Settings in sftpgo.json:
```
  "httpd": {
    "bindings": [
      {
        "enabled_login_methods": 0,
 ```

**Reply by fdb3** [reply]
thank you very much for your answer. This is very helpful!
